environment.yml

```diff
@@ -82,7 +81,7 @@ dependencies:
   - flake8-bugbear
   - git
   - setuptools_scm
-  - openssl<3.0.0 # Pin until openssl issue is resolved by conda
+  - openssl
```

Comment:
I assume this means they resolved the issue?

Reply:
Yes, I tested it unpinned, and it worked in both a Python 3.11 and 3.10 environments.